### PR TITLE
Use None_ for things named None

### PR DESCRIPTION
**Pull request by altendky**

### CODE REVIEW

File: PyQt5-stubs/QtWidgets.pyi [hunk 1]
```diff
@@ -1038,7 +1038,7 @@ class QAction(QtCore.QObject):
 class QActionGroup(QtCore.QObject):
 
     class ExclusionPolicy(int): ...
-    # None = ... # type: 'QActionGroup.ExclusionPolicy'
+    None_ = ... # type: 'QActionGroup.ExclusionPolicy'
```
Comment (BryceBeagle) [inline PyQt5-stubs/QtWidgets.pyi:1041]: I think this is an upstream bug.

```pycon
>>> QActionGroup.ExclusionPolicy.None_
```
```pytb
AttributeError: None_
```
but this works
```pycon
>>> getattr(QActionGroup.ExclusionPolicy, "None")
```
```
<ExclusionPolicy.None: 0>
```

Comment (BryceBeagle) [reply PyQt5-stubs/QtWidgets.pyi:1041]: Same with `QOcspRevocationReason.None`

**Review by BryceBeagle — CHANGES_REQUESTED** [review]

File: PyQt5-stubs/QtWidgets.pyi (hunk 1 above)
Comment (altendky) [reply PyQt5-stubs/QtWidgets.pyi:1041]: Mmm, I guess leaving it commented makes more sense than lying.  As I recall there is `None_` elsewhere in upstream.  Thanks.  I still need to get my head into stubs and figure out how to test them.  I did see the other ticket about the pytest plugin and also saw another plugin.

Comment (BryceBeagle) [reply PyQt5-stubs/QtWidgets.pyi:1041]: I'll reach out on the mailing list about these

**Comment by BryceBeagle** [comment]
This should be fixed now in the [upstream nightly](https://www.riverbankcomputing.com/static/Downloads/PyQt5/ChangeLog-5.15.2.dev2009241214)
```
2020-09-24  Phil Thompson  <phil@riverbankcomputing.com>

	* PyQt5.msp:
	Fixed QActionGroup.ExclusionPolicy.None and
	QOcspRevocationReason.None.
	[cdcf3488537b] [tip] <5.15-maint>
```

**Pull request closed without merging**